### PR TITLE
Ports TG pipes

### LINKED ISSUE #704: Cryo cells in medbay not working, and related missing items.
## Reproduction:
When medbay was re-done the cryo cells were not given oxygen canisters or a freezer, they wont work when I click on them, and the table with cryoxadone beakers no longer start with 4, but only 3.

### PR DESCRIPTION
**Pull request by spookydonut**
:cl: 
add: ported TG pipe code
/:cl:

[why]: # needed for maploader to init pipes properly

To fix:
- [x] Instant vent exit
- [x] Can't exit from broken pipes

Fixes #704

### CODE REVIEW

File: code/_onclick/ventcrawl.dm [hunk 1]
```diff
@@ -45,77 +47,88 @@ var/list/ventcrawl_machinery = list(/obj/machinery/atmospherics/unary/vent_pump,
...
+			visible_message("<span class='notice'>[src] begins climbing into the ventilation system...</span>" ,"<span class='notice'>You begin climbing into the ventilation system...</span>")
+
+			if(!do_after(src, 45, FALSE, 5, BUSY_ICON_GENERIC))
+				return
+
+			if(!client)
+				return
+
+			if(iscarbon(src) && can_ventcrawl())//It must have atleast been 1 to get this far
+				var/failed = 0
```
Comment (Rohesie) [inline code/_onclick/ventcrawl.dm]:
```suggestion
				var/failed = FALSE
```

File: code/_onclick/ventcrawl.dm [hunk 2]
```diff
@@ -45,77 +47,88 @@ var/list/ventcrawl_machinery = list(/obj/machinery/atmospherics/unary/vent_pump,
...
+		if(vent_found_parent && (vent_found_parent.members.len || vent_found_parent.other_atmosmch))
+			visible_message("<span class='notice'>[src] begins climbing into the ventilation system...</span>" ,"<span class='notice'>You begin climbing into the ventilation system...</span>")
+
+			if(!do_after(src, 45, FALSE, 5, BUSY_ICON_GENERIC))
+				return
+
+			if(!client)
+				return
+
+			if(iscarbon(src) && can_ventcrawl())//It must have atleast been 1 to get this far
```
Comment (Rohesie) [inline code/_onclick/ventcrawl.dm]:
```suggestion
			if(can_ventcrawl())//It must have atleast been 1 to get this far
```
The carbon check shouldn't be necessary, as it's covered in the other one.

File: code/_onclick/ventcrawl.dm [hunk 3]
```diff
@@ -45,77 +47,88 @@ var/list/ventcrawl_machinery = list(/obj/machinery/atmospherics/unary/vent_pump,
...
+				var/failed = 0
+				var/list/items_list = get_equipped_items() //include_pockets = TRUE)
+				if(items_list.len)
+					failed = 1
```
Comment (Rohesie) [inline code/_onclick/ventcrawl.dm]:
```suggestion
					failed = TRUE
```

File: code/_onclick/ventcrawl.dm [hunk 4]
```diff
@@ -45,77 +47,88 @@ var/list/ventcrawl_machinery = list(/obj/machinery/atmospherics/unary/vent_pump,
...
+				if(failed)
+					to_chat(src, "<span class='warning'>You can't crawl around in the ventilation ducts with items!</span>")
+					return
+
+			if(!ventcrawl_carry())
+				return
+
+			visible_message("<span class='notice'>[src] scrambles into the ventilation ducts!</span>","<span class='notice'>You climb into the ventilation ducts.</span>")
+			if(!istype(src,/mob/living/carbon/Xenomorph/Hunter)) //Hunters silently enter/exit vents.
```
Comment (Rohesie) [inline code/_onclick/ventcrawl.dm]:
```suggestion
			if(!isxenohunter(src)) //Hunters silently enter/exit vents.
```

File: code/_onclick/ventcrawl.dm (hunk 2 above)
Comment (spookydonut) [reply code/_onclick/ventcrawl.dm]: no because simple_animals arent carbon

**Pull request merged**